src/main/grammars/ElmParser.bnf

```diff
@@ -138,11 +139,9 @@ external DotWithoutTrailingWhitespace ::= parseDotWithoutTrailingWhitespace
 external DotWithoutLeadingWhitespace ::= parseDotWithoutLeadingWhitespace
 external MinusWithoutTrailingWhitespace ::= parseMinusWithoutTrailingWhitespace
 
-UpperCaseQID ::= UPPER_CASE_IDENTIFIER (DotWithoutWhitespace UPPER_CASE_IDENTIFIER)* !DotWithoutLeadingWhitespace
```

Comment:
I removed the `!DotWithoutLeadingWhitespace` lookahead because it no longer seems to be necessary (probably due to the grammar changes that I made earlier today).